app/models/katello/glue/pulp/pulp_content_unit.rb

```diff
@@ -17,7 +17,11 @@ def backend_data
 
     module ClassMethods
       def unit_handler
-        Katello.pulp_server.extensions.send(self.name.demodulize.pluralize.underscore)
+        if self.name == "Katello::PackageGroup"
+          Katello.pulp_server.extensions.send(self.name.demodulize.underscore)
+        else
+          Katello.pulp_server.extensions.send(self.name.demodulize.pluralize.underscore)
+        end
```

Comment:
I modified this in my PR: https://github.com/Katello/katello/pull/5326/files#diff-602019667939c0e00e63eec1b276c044L19

and overrode it for errata:  https://github.com/Katello/katello/pull/5326/files#diff-1739022428b7fe94d6a65929f3f453a1L95

I think that is a better solution as errata is the outlier here, the the rest should be the same.